### PR TITLE
Minor formatting fixes in dom-repeat's doc comment.

### LINKED ISSUE #4727: Doc comment for dom-repeat has typo and inconsistent formatting
### Description

The doc comment for the `<dom-repeat>` element has two very minor issues:

1. A typo: `< /script>` (with an extra space) instead of `</script>` in the example code (introduced in commit 88756fe)
2. Inconsistent formatting: most of the code uses 2 spaces to indent, but a few parts use 4 spaces

#### Live Demo

https://www.webcomponents.org/element/polymer/Polymer/v2.0.1/elements/Polymer.DomRepeat

#### Steps to Reproduce

N/A

#### Expected Results

1. Closing HTML tags in example code should not contain spaces
2. Code samples should be consistently indented with 2 spaces

#### Actual Results

1. Example HTML code has an extra space in the closing `</script>` tag (`< /script>`)
2. Code samples have inconsistent indentation, with a mix of 2 spaces and 4 spaces

### Browsers Affected

N/A

### Versions

- Polymer: v2.0.0-rc.4+
- webcomponents: N/A

### PR DESCRIPTION
**Pull request by acdvorak**
### Reference Issue

Fixes #4727

### CODE REVIEW

**Comment by arthurevans** [comment]
I believe the `< /script>` is intentional to avoid issues with the doc tooling.

File: lib/elements/dom-repeat.html [hunk 1]
```diff
@@ -53,13 +53,13 @@
...
-   *             {first: 'Bob', last: 'Smith'},
-   *             {first: 'Sally', last: 'Johnson'},
-   *             ...
+   *           {first: 'Bob', last: 'Smith'},
+   *           {first: 'Sally', last: 'Johnson'},
+   *           ...
    *         ];
    *       }
    *     });
-   *   < /script>
```
Comment (azakus) [inline lib/elements/dom-repeat.html]: Please leave the space in the `script`. Otherwise the `</script>` will break this module by ending the script early.

**Comment by TimvdLippe** [comment]
Friendly ping to process the review feedback :)

**Comment by TimvdLippe** [comment]
I am closing this PR as it appears to be stale. Feel free to reopen once the feedback has been processed and the PR has been updated!

**Pull request closed without merging**